### DISCUSSION TITLE
TP 4 - Ejercicio 1

### DISCUSSION THREAD

**Discussion by Clirussa** [INGCOM-UNRN-P1/cursada-2023#130] [Preguntas y respuestas] [answered]
Hola, con @agosgit  tenemos este error que no entendemos como solucionarlo o que está mal. 

Esperamos ayuda, 
Saludos
Agos y Day

![puntero- error linea 18](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/102642611/e207ad3d-cff1-4f62-8822-fa5abe697f8b)

**Comment by martinvilu** [comment]
¿Y que esta mal?

**Reply by AgostinaApraiz** [reply]
Hola! Que tal? Estos son los errores que tira.
Muchas gracias
![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/135774264/f1016bb9-7318-4245-ad61-ab5cc74a7787)

**Comment by Clirussa** [comment]
Nos tira errores diferentes, con el mismo código. Este es el error por el que habíamos preguntado. 

![error-de- ejercicio de punteros](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/102642611/a59c58d3-556c-4e59-b5d9-404d65d00dc2)

**Comment by martinvilu** [comment, marked as answer]
Si han creado un carácter(`char`), el puntero debe ser a carácter; no a `int`.

Por eso es 'incompatible', porque realmente lo es; a pesar de que los punteros tienen todos el mismo tamaño, a lo que apuntan no.

**Comment by Clirussa** [comment]
Very good! gracias!!
